docs/lang/attributes.md

```diff
@@ -69,15 +69,23 @@ It has two meanings:
    assignments to the use the ports. See the documentation on [externalize][] for more information.
 2. If the cell is a memory and has an `external` attribute on it, the Verilog backend (`-b verilog`) generates code to read `<cell_name>.dat` to initialize the memory state and dumps out its final value after execution.
 
-### `static(n)`
```

Comment:
I think we should keep this and mark it as deprecated (and link to the new `promotable` and `interval`) attributes